pom.xml

```diff
@@ -169,6 +169,7 @@
                             <includes>.*</includes>
                             <excludes></excludes>
                             <!--<inputSchema></inputSchema>--> <!-- This will generate all schema of db, better to only generate the one interested in -->
+                            <inputSchema></inputSchema>
```

Comment:
Check jooq documentation but you can only have one <inputSchema>
I already defined  <inputSchema>information_schema</inputSchema> which is only what is needed by default

Reply:
removed